<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="newAction"/>
    <addaction name="saveAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="newAction">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>另存为(&amp;S)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
